aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -217,6 +217,30 @@ def to_dict(self, origin: Optional[str]) -> Dict[str, str]:
             headers["Access-Control-Allow-Credentials"] = "true"
         return headers
 
+    @staticmethod
+    def extract_origin_header(resolver_headers: Dict):
```

Comment:
```suggestion
    def extract_origin_header(resolver_headers: Mapping[str, Union[str, list[str]]) -> str:
```

Reply:
We are now using the `get_header_value` function and there is no overload for this signature. In the end this is a Dict, so I think we can keep this signature.